packages/xo-web/src/index.scss

```diff
@@ -213,6 +213,13 @@ $select-input-height: 40px; // Bootstrap input height
   border: 1px solid black;
   margin: 5px 10px;
   width: 250px;
+  // Workaround to prevent ".input-group > .form-control" and ".input-group > .input-group-btn > .btn" from hiding the notifications.
+  // (https://github.com/vatesfr/xen-orchestra/issues/3874)
+  // In bootstrap, ".input-group > .form-control" and ".input-group > .input-group-btn > .btn"
```

Comment:
See [this](https://github.com/vatesfr/xen-orchestra/pull/4305#discussion_r302451181). It's `.input-group .form-control`, not `.input-group > .form-control`.